index.html

```diff
@@ -12,27 +12,37 @@
 
 <body>
     <main>
+        <header>
+            <nav>
+                <a href="#headline" id="gabriel">Gabriel</a>
+                <ul class="tab tab-nav">
+                    <li class="tab-link">
+                        <a href="#portfolio">Portfolio</a>
+                    </li>
+                    <li class="tab-link">
+                        <a href="#about">About</a>
+                    </li>
+                    <li class="tab-link">
+                        <a href="#contact">Contact</a>
+                    </li>
+                </ul>
+                <div class="hamburger ham" id="hamburger">
+                    <div class="bar-1"></div>
+                    <div class="bar-2"></div>
+                    <div class="bar-3"></div>
+                </div>
+            </nav>
+            <nav class="curtain curtain-anim" id="curtain">
+                <ul class="tab-items">
+                    <li id="nav1"><a href="#portfolio" class="mobile-links">Portfolio</a></li>
+                    <li id="nav2"><a href="#about" class="mobile-links">About</a></li>
+                    <li id="nav3"><a href="#contact" class="mobile-links">Contact</a></li>
+                </ul>
+            </nav>
```

Comment:
- [ ] your mobile menu looks fantastic 💯 , kindly move the links to the top of the menu to match the Figma design: 

Your Design         | Figma Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/89282221/190705929-d3d5e681-e153-4a79-a82b-bb82ad77eec9.png) | ![image](https://user-images.githubusercontent.com/89282221/190705984-c7cde683-d735-4bf0-8a76-10af0a76e5b1.png)